### PR TITLE
Copy Jepsen tests for Cassandra and Scalar DB

### PR DESCRIPTION
**Pull request by yito88**
Just copy files from `scalar-labs/jepsen/cassandra` and `scalar-labs/scalardb/jepsen/scalardb`

I've checked if I can run tests with Jepsen docker in https://github.com/jepsen-io/jepsen.
I wrote how to run in the new `README.md`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# cd /scalar-jepsen/cassandra
+# lein run test --test lwt
+```
+
+  - Check README in each test for more detail
+
+# Run tests without Docker
+1. Launch debian machines as a control machine and Cassandra nodes
+  - We recommend 1 control machine and 5 node's machines
+    - You can decrease the number of nodes. If it's decreased, you need to specify the nodes when a test starts.
```
Comment (siyopao) [inline README.md]:
```suggestion
    - You can decrease the number of nodes. If you do so then you will need to specify the nodes when starting a test.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+```
+# cd /scalar-jepsen/cassandra
+# lein run test --test lwt
+```
+
+  - Check README in each test for more detail
+
+# Run tests without Docker
+1. Launch debian machines as a control machine and Cassandra nodes
+  - We recommend 1 control machine and 5 node's machines
```
Comment (siyopao) [inline README.md]:
```suggestion
  - We recommend 1 control machine and 5 node machines
```

File: scalardb/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Jepsen tests for Scalar DB
```
Comment (siyopao) [inline scalardb/README.md]: This needs to be updated I think.

File: README.md (hunk 1 above)
Comment (yito88) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (yito88) [reply README.md]: Fixed

File: scalardb/README.md (hunk 3 above)
Comment (yito88) [reply scalardb/README.md]: Updated

Comment (siyopao) [reply scalardb/README.md]: Maybe for number 1 you can say something like:

1. Start the Docker nodes and log into jepsen-control (as explained [here](../README.md)).

File: README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+
+# Run tests with Jepsen Docker
+1. Clone [Jepsen repository](https://github.com/jepsen-io/jepsen)
+2. Configure to mount `scalar-jepsen` directory by editing `${JEPSEN}/docker/docker-compose.dev.yml` like the following
+
+```yaml
+   control:
+     volumes:
+       - ${JEPSEN_ROOT}:/jepsen # Mounts $JEPSEN_ROOT on host to /jepsen control container
```
[the remark this answers is not in the document]
Comment (yito88) [reply README.md]: `${JEPSEN_ROOT}` is defined by another file.
I changed the name into the consistent name `${JEPSEN_ROOT}`.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  - We recommend 1 control machine and 5 node machines
+    - You can decrease the number of nodes. If you do so then you will need to specify the nodes when starting a test.
+2. Install Java8 on each machine
+```sh
+sudo apt install openjdk-8-jre
+```
+
+3. Install Leiningen (https://leiningen.org/) on the control machine
+4. Make a SSH key pair on the control machine
+5. Add the public key to login nodes as root on each node
```
[the remark this answers is not in the document]
Comment (yito88) [reply README.md]: It's for Jepsen to login nodes. I modified the sentence.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+
+# Run tests with Jepsen Docker
+1. Clone [Jepsen repository](https://github.com/jepsen-io/jepsen)
+2. Configure to mount `scalar-jepsen` directory by editing `${JEPSEN_ROOT}/docker/docker-compose.dev.yml` like the following
+
+```yaml
+   control:
+     volumes:
+       - ${JEPSEN_ROOT}:/jepsen # Mounts $JEPSEN_ROOT on host to /jepsen control container
+       - ${SCALAR_JEPSEN}:/scalar-jepsen
```
Comment (feeblefakie) [inline README.md]: `${SCALAR_JEPSEN_ROOT}` for consistency ?

**Pull request merged**